### PR TITLE
Study burst info in timeline

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: src/test/java/org/sagebionetworks/bridge/models/schedules2/timelines/ScheduledSessionTest.java [hunk 1]
```diff
@@ -116,5 +117,25 @@ public void copyWithoutAssessments() {
...
+        session.setGuid(SESSION_GUID_1);
+        
+        TimeWindow window = new TimeWindow();
+        window.setGuid(SESSION_WINDOW_GUID_1);
+        
+        ScheduledSession schSession = new ScheduledSession.Builder()
+                .withSession(session)
+                .withTimeWindow(window)
+                .withStartEventId("study_burst:foo:01")
+                .build();
```
Comment (alxdarksage) [inline src/test/java/org/sagebionetworks/bridge/models/schedules2/timelines/ScheduledSessionTest.java:136]: Does this test what you think it tests?

Comment (alxdarksage) [reply src/test/java/org/sagebionetworks/bridge/models/schedules2/timelines/ScheduledSessionTest.java:136]: Yes, this tests that the builder correctly parses that event ID into studyBurstId and studyBurstNum.

**Review by jnelsonjava — APPROVED** [review]
Looks good!

**Review by nategbrown9 — APPROVED** [review]
Looks good.

**Pull request merged**